README.md

```diff
@@ -27,17 +27,37 @@ birthdays:
   - name: Elvis
     date_of_birth: 1935-01-08
     icon: 'mdi:music'
-~~~~
+  - unique_id: bond_james_bond
+    name: James Bond
+    date_of_birth: 1920-05-25
+    icon: 'mdi:pistol'
+    attributes:
+      occupation: "Agent"
+      license_to_kill: "Yes"
+  - unique_id: einstein
+    name: 'Albert Einstein'
+    date_of_birth: 1879-03-14
+    icon: 'mdi:lightbulb-on'
+    attributes:
+      occupation: 'Theoretical physicist'
+      iq: 'Genius level'
+      sense_of_humor: 'Einsteinian'
+```
 Restart homeassistant
 
 ## Entities
 All entities are exposed using the format `birthdays.{name}`. Any character that does not fit the pattern `a-z`, `A-Z`, `0-9`, or `_` will be changed. For instance `Frodo Baggins` will get entity_id `frodo_baggins`, and Swedish names like [`Sven-Göran Eriksson`](https://sv.wikipedia.org/wiki/Sven-G%C3%B6ran_Eriksson) will get entity_id `sven_goran_eriksson`.
 
+## Custom attributes
+You can add custom attributes to each birthday, for instance to add an icon or other metadata.
+To do this, add a dictionary under the `attributes` key in the configuration (see example above). The dictionary can contain any key-value pairs you want, and will be exposed as attributes on the entity.
+Fetching the attributes can be done using the `attributes` property in a template, for instance `{{ states.birthdays.einstein.attributes.occupation }}` will return `Theoretical physicist`.
```

Comment:
`{{ states.birthdays.einstein.attributes.occupation }}` -> this should be changed according to HA: `{{ state_attr('birthdays.einstein', 'occupation') }}`, otherwise it can lead to lots of error logs (been there 😅  )

Reply:
I actually wasn't aware of that, but thanks for the heads up! 😁